reflex/components/radix/themes/typography/link.py

```diff
@@ -21,8 +21,10 @@
 
 LiteralLinkUnderline = Literal["auto", "hover", "always"]
 
+from reflex.components.el.elements.metadata import Link as BaseLink
```

Comment:
```suggestion
from reflex.components.el.elements.inline import A
```

The HTML `Link` element is used for metadata stuff, this radix `Link` element needs to extend from the `A` (anchor) element.